### ISSUE TITLE
Can't import content by folder

### ISSUE THREAD

**Issue by Daripa** [marcrobledo/retroarch-playlist-editor#17]
Can only import individual files not entire folders.
I'm using android and the drag and drop feature is not applicable.

**Comment by porg** [comment]
Please make your proposal more specific: Folders with what content?
1. With multiple playlists? What would be the purpose? Merging all those playlists into one playlist?
2. What if your folder contains non-playlist-files? How shall it behave then?
3. What if that folder contains (a) further subfolder(s) and these again? How far down (how recursive) shall it be?

AFAIR there is no standard for a true folder-upload in web browsers, just upload of single or multiple files. Maybe some browsers (on Android that you are using) emulate folder uploading, by translating dropping a folder into the upload area into an upload of all files in that folder (and of subfolders, sub-sub-folders, …?) into a list of multiple files.

The request in its current length & specificity seems not too thoughtful or at least not recognizible as such to others.

**Comment by marcrobledo** [comment]
Yes, it is possible for web browser to choose an entire folder (and even get a full list of its files and their relative paths) with `webkitdirectory`.

However, you should be able to choose various files by long-tapping then checking the desired files. Depending on the mobile OS and its version, you might even have a choose all files button in the toolbar.

This coud be implemented on Retroarch Playlist Editor, but I am not sure if it is really needed, I figure almost all Android (and probably iOS too) versions allow you to select all files in a folder.

**Comment by porg** [comment]
- Thanks for info on what's technically feasible.
- Still I asked the OP for the purpose (merging?) and also how to deal with the pitfalls (non playlist files, the potential recursive nature of this).
- Until then this issue will only be on standby I assume.
